src/sagemaker/config/config_schema.py

```diff
@@ -135,6 +136,9 @@ def _simple_path(*args: str):
 ENDPOINT_CONFIG_ASYNC_KMS_KEY_ID_PATH = _simple_path(
```

Comment:
config injection that uses ENDPOINT_CONFIG_ASYNC_INFERENCE_PATH will also plug in this new value. Can we add or update unit tests for model.py and for session.py? Existing unit tests should have `_with_sagemaker_config_injection` in the name. Might be able to just add to the existing ones.